src/change_log.js

```diff
@@ -1,6 +1,10 @@
 const core = require('@actions/core');
 const commitParser = require('conventional-commits-parser');
-const { OPEN_AI_IMPROVE_CHANGELOG_REQUEST, openai } = require('./open_ai');
+
+const {
+    OPEN_AI_IMPROVE_CHANGELOG_REQUEST,
+    OPEN_AI_IMPROVE_CHANGELOG_ROLE_DEFINITION,
+    openai } = require('./open_ai');
```

Comment:
Does the new ESLint config format it like this? I thought it should format it like this:
```suggestion
const {
    OPEN_AI_IMPROVE_CHANGELOG_REQUEST,
    OPEN_AI_IMPROVE_CHANGELOG_ROLE_DEFINITION,
    openai,
} = require('./open_ai');
```